backend/automated_scraping_service.py

```
@@ -0,0 +1,242 @@
+    """Run an immediate scrape (for testing/manual runs)"""
+    db_path = os.path.join(os.path.dirname(__file__), 'instance', 'opportunities.db')
+    service = AutomatedScrapingService(db_path)
+    
+    print("🚀 Running immediate scrape...")
+    results = service.run_scheduled_scrape()
+    
+    print(f"\n✅ Immediate scrape complete!")
+    print(f"Added {results.get('total_new', 0)} new opportunities")
+
+if __name__ == "__main__":
+    import sys
```

Flake8 (7.2.0)

[error] redefinition of unused 'sys' from line 8

(F811)